tests/shape_poly_test.py

```diff
@@ -94,19 +94,20 @@ def eval(d: shape_poly.DimSize):
         f"{expected_sym=} {expected_concrete=} {compute_concrete=} {env=}")
 
   def test_parse_shape(self):
```

Comment:
Doesn't this duplicate `test_parse_shape` in jax/experimental/jax2tf/tests/shape_poly_test.py?